examples/oidc/css-api/nextjs/README.md

```diff
@@ -0,0 +1,43 @@
+# Service Account with Nextjs App
+
+This folder contains an example application using the CSS API, built with nextjs.
+
+## Getting Started
+
+### Prerequisites
+
+Ensure to have npm and nodejs installed, with a version of node >= 18.0.0.
+
+### Environment variable configuration
+
+To run this demo, you will require a CSS API account. From the CSS Application, navigate to to **My Dashboard**, and select the **My Teams** tab. If you do not have a team, you can create one. Once you have a team, select the CSS API Account tab below and create an API account.
+
+**Note**: _While not required, you may want to add an integration or two to the team to be able to see them in the demo application._
+
+With the created account you can now click the download icon to get your credentials. Copy them into a .env file, following [.env.example](./.env.example), adding the values from the downloaded json as below:
+
+```
+SSO_CLIENT_ID=<clientId>
+SSO_CLIENT_SECRET=<clientSecret>
+SSO_API_URL=<tokenUrl>
```

Comment:
@jlangy I would keep it consistent so better if we call SSO_TOKEN_URL? let me know your thoughts

Reply:
Sounds good, I'll switch that